.env.default

```diff
@@ -1,7 +1,7 @@
-AWS_ACCESS_KEY_ID=dummy-123
-AWS_SECRET_ACCESS_KEY=dummy-123
+AWS_ACCESS_KEY_ID=dummy123
+AWS_SECRET_ACCESS_KEY=dummy123
 AWS_ENDPOINT_URL=http://localhost:8080
 AWS_DEFAULT_REGION=eu-central-1
 AWS_DYNAMODB_TABLE_NAME=test-db
-ALLOWED_DOMAINS=.*localhost,.*admin\.ch,.*bgdi\.ch
+ALLOWED_DOMAINS=.*localhost((:[0-9]*)?|\/)?$,.*admin\.ch$,.*bgdi\.ch$
```

Comment:
I don't feel it's very intuitive to add `$` to all domains in a configuration, I would expect this behaviour without explicitly adding it. I think using `re.fullmatch` when checking the string should achieve the same result.